### ISSUE TITLE
gcovr + jinja2 3.1.0

### ISSUE THREAD

**Issue by mcsakoff** [gcovr/gcovr#577]
Jinja2 3.1.0 released Mar 24, 2022:
```
  File "/usr/local/lib/python3.8/site-packages/gcovr/writer/html.py", line 159, in highlighter_for_file
    from jinja2 import Markup
ImportError: cannot import name 'Markup' from 'jinja2' (/usr/local/lib64/python3.8/site-packages/jinja2/__init__.py)
```

Jinja doesn't do `from .utils import Markup` anymore in top level `__init__.py` file.

**Comment by Spacetown** [comment]
Already fixed on master with #576.

**Comment by latk** [comment]
Temporary workaround is to manually install an older jinja2 version before installing gcovr:

```bash
pip install 'jinja2<3.1' gcovr
```

or to install the gcovr development version:

```bash
pip install git+https://github.com/gcovr/gcovr
```
